doc/sphinxext/sphinx_gallery/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 """
 import os
-__version__ = '0.1.3'
+__version__ = '0.1.4-dev'
```

Comment:
Strange, the changes to gen_rst.py are missing from this commit, which implies that the download buttons don't render right. I'll merge and add them manually. Thanks